standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.java

```diff
@@ -2587,19 +2587,24 @@ public NotificationEventResponse getNextNotification(long lastEventId, int maxEv
     rqst.setMaxEvents(maxEvents);
     NotificationEventResponse rsp = client.get_next_notification(rqst);
     LOG.debug("Got back " + rsp.getEventsSize() + " events");
```

Comment:
if response can be null ..this log can cause exception

Reply:
rsp won't be null.